x/ethbridge/types/claim_type.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+// ClaimType is an enum used to represent the type of claim
+type ClaimType int
+
+const (
+	LockText = ClaimType(iota)
+	BurnText
+)
+
+var ClaimTypeToString = [...]string{"lock", "burn"}
+var StringToClaimType = map[string]ClaimType{
```

Comment:
We ideally want to return an error, not a panic, so that the error can be shown in the HTTP request and we don't crash the HTTP server?

changed to that here: https://github.com/cosmos/peggy/pull/81/commits/deb9746a2fee9d423d38cf5aca517cf216afd204